sdk/eventhub/event-hubs/src/sender.ts

```diff
@@ -42,26 +42,27 @@ export class EventSender {
   constructor(context: ConnectionContext, options?: EventSenderOptions) {
     this._context = context;
     this._senderOptions = options || {};
-    const partitionId = this._senderOptions.partitionId != undefined ? String(this._senderOptions.partitionId) : undefined;
+    const partitionId =
+      this._senderOptions.partitionId != undefined ? String(this._senderOptions.partitionId) : undefined;
     this._eventHubSender = EventHubSender.create(this._context, partitionId);
   }
 
   /**
    * Send a batch of EventData to the EventHub using the options provided.
    *
-   * @param events  An array of EventData objects to be sent in a Batch message.
+   * @param eventData  An individual EventData or array of EventData objects to be sent in a Batch message.
```

Comment:
> An individual EventData or array of EventData objects to be sent. If an array is provided, the individual events are batched into a single AMQP message that gets sent over the network. This can fail if the size of the final message exceeds the max limit.

We can update the error name and limit later